tools/mo/openvino/tools/mo/utils/version.py

```diff
@@ -75,3 +73,15 @@ def get_simplified_ie_version(env=dict(), version=None):
     if m and len(m.groups()) == 3:
         return simplify_version(m.group(3))
     return simplify_version(version)
+
+
+def extract_version_hash(full_version: str):
+    """
+    extracts 11 character hash from the full version string
```

Comment:
please correct description

Reply:
I made hash length an optional argument of the function. Prototype is self-explanatory, removed description.

Reply:
Changed a bit more this function. Definition is still self-explanatory